selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/WorkspacesListTest.java

```diff
@@ -338,7 +338,6 @@ public void checkWorkspaceActions() throws Exception {
     newWorkspace.selectStack(BLANK);
     newWorkspace.clickOnCreateButtonAndEditWorkspace();
     workspaceOverview.checkNameWorkspace(NEWEST_CREATED_WORKSPACE_NAME);
-    dashboard.waitWorkspacesCountInWorkspacesItem(getWorkspacesCount());
```

Comment:
How about checking presence of workspace at server side? I mean getting workspace DTO by its name from WS master.